apisix/plugins/sls-logger.lua

```diff
@@ -126,26 +127,23 @@ local function remove_stale_objects(premature)
 end
 
 local function combine_syslog(entries)
-    local data
+    local data = {}
```

Comment:
how about the name `items`?

Reply:
Good, I am tangled what name for the local var.

Reply:
fixed.